parsing/parchmint-parser.js

```diff
@@ -146,6 +181,10 @@ class ParchmintParser {
         this.connFeatures = new Map();
 
         this.schemaValidator = ajv.compile(schema);
+
+        this.maxX = 0;
+        this.maxY = 0;
+        this.maxPad = 50;
```

Comment:
Do these values get set somewhere within the system? Based on the function using the `maxX` and `maxY` values below it seems like it must get set to be the maximum x/y values based on the components and/or connections. I'm find with finding these values and using them until we plug the hole with ParchMint now having a total size but we will definitely need the `maxPad` to be configurable from the website and probably will also want the `maxX` and `maxY` to be configurable as well (something like tries to figure out the max x/y unless one is provided).

Reply:
Yes right now ```maxX``` and ```maxY``` are set to the farthest component/channel position when we parse the ParchMint. I'll add them to the config file.

Reply:
Yeah add them to the config file. This is probably something we will want to be able to set from the website until we have it patched into the ParchMint standard but that will come down the line.

Reply:
They're in the config and can be inputted from the web page.